### PR TITLE
[SDPA-614] Added LatestNewsList component.

### PR DESCRIPTION
**Pull request by alan-cole**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-614

### Changed

* Updated CardContent to support inline images.
* Added CardImageNavigation to Card component.
* Updated CardNavigationFeatured to support date and topic.
* Added LatestNewsList component.
* Updated docs / tests.

### Screenshots

screen shot 2018-09-06 at 11 32 52 am

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "@dpc-sdp/ripple-grid": "file:packages/Atoms/Grid",
     "@dpc-sdp/ripple-hero-banner": "file:packages/Organisms/HeroBanner",
     "@dpc-sdp/ripple-icon": "file:packages/Atoms/Icon",
+    "@dpc-sdp/ripple-latest-news-list": "file:packages/Organisms/LatestNewsList",
```
Comment (tim-yao) [inline package.json]: This is not "latest news list" component, should be "featured news" component.
I will prefer to have a "ripple-news" package for hosting it.

Comment (alan-cole) [reply package.json]: screen shot 2018-11-07 at 11 57 26 am

Can we please get these details on the ticket before development?

**Comment by alan-cole** [comment]
Closing as this branch has become out of date. Opening a new branch to continue development.

**Comment by alan-cole** [comment]
New PR At: https://github.com/dpc-sdp/ripple/pull/117

**Pull request closed without merging**